scripts/rollup/build.js

```diff
@@ -357,7 +357,7 @@ function getPlugins(
     bundleType === RN_FB_DEV ||
     bundleType === RN_FB_PROD ||
     bundleType === RN_FB_PROFILING;
-  const shouldStayReadable = isFBBundle || isRNBundle || forcePrettyOutput;
+  const shouldStayReadable = isFBWWWBundle || isRNBundle || forcePrettyOutput;
```

Comment:
Arguably `shouldStayReadable` should be false for non-FB since we don't trust the minifier.